neo/Consensus/ConsensusService.cs

```diff
@@ -18,6 +18,8 @@ namespace Neo.Consensus
 {
     public sealed class ConsensusService : UntypedActor
     {
+        public const int MaxValidatorsCount = byte.MaxValue;
```

Comment:
https://github.com/neo-project/neo/blob/7cf286941f7f5541e2ab4bf182583452eec8fc5a/neo/Ledger/Blockchain.cs#L30

Can use `Blockchain.MaxValidators` directly.

Reply:
Yes. Should I make the change, or are you working on it?